src/fixes/extra_blank_line.rs

```diff
@@ -0,0 +1,179 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct ExtraBlankLineFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for ExtraBlankLineFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "ExtraBlankLine",
+        }
+    }
+}
+
+impl Fix for ExtraBlankLineFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_warnings(
+        &self,
+        warnings: Vec<&mut Warning>,
+        lines: &mut Vec<LineEntry>,
+    ) -> Option<usize> {
+        let warning_count = warnings.len();
+
+        // check and remove all blank lines.
+        lines.dedup_by(|a, b| a.is_empty() && b.is_empty());
+
+        // mark as fixed
+        for warning in warnings {
+            warning.mark_as_fixed();
+        }
+
+        Some(warning_count)
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::path::PathBuf;
+
+    #[test]
+    fn fix_multiple_lines_test() {
+        let fixer = ExtraBlankLineFixer::default();
+        let file = FileEntry {
+            path: PathBuf::from(".env"),
+            file_name: ".env".to_string(),
+            total_lines: 3,
+        };
+
+        let warnings = vec![];
+        let lines = vec![
+            LineEntry {
+                number: 1,
+                file: file.clone(),
+                raw_string: String::from("FOO=BAR"),
+            },
+            LineEntry {
+                number: 2,
+                file: file.clone(),
+                raw_string: String::from(""),
+            },
+            LineEntry {
+                number: 3,
+                file: file.clone(),
+                raw_string: String::from("HOGE=HUGA"),
+            },
+        ];
+        let mut fixing_lines = lines.clone();
+
+        assert_eq!(
+            Some(0 as usize),
+            fixer.fix_warnings(warnings, &mut fixing_lines)
+        );
+        assert_eq!(lines, fixing_lines);
+    }
+
+    #[test]
+    fn fix_multiple_lines_test_double() {
```

Comment:
```suggestion
    fn fix_one_extra_blank_line_test() {
```
It will be more obvious